geom/opticks_raindrop.gdml

```diff
@@ -109,16 +122,16 @@
       <fraction n="0.524" ref="Pb2080x560299383420"/>
     </element>
     <material name="G4_Pb0x5602993831c0" state="solid">
-          <property name="RINDEX" ref="RINDEX0x560299349070"/>
-
+          <property name="RINDEX" ref="RINDEX0x5602993a1df2"/>
+<property name="GROUPVEL" ref="GROUPVEL0x56029938bb90"/>
       <T unit="K" value="293.15"/>
       <MEE unit="eV" value="823"/>
       <D unit="g/cm3" value="11.35"/>
       <fraction n="1" ref="Pb0x560299349da0"/>
     </material>
     <material name="VACUUM0x5602990cb360" state="gas">
-      <property name="RINDEX" ref="RINDEX0x5602993a1df0"/>
-
+      <property name="RINDEX" ref="RINDEX0x5602993a1df3"/>
+<property name="GROUPVEL" ref="GROUPVEL0x56029938bb90"/>
```

Comment:
Are these new lines needed?

Reply:
Rindex is needed for all materials. I have tried running without refractive index before and there were no hits. I am not sure if GROUPVEL is needed or not, some GDML uses it, some not.